packages/js/product-editor/src/blocks/schedule-sale/edit.tsx

```diff
@@ -145,11 +159,13 @@ export function Edit( {}: BlockEditProps< ScheduleSalePricingBlockAttributes > )
 								dateOnSaleFromGmtValidationError && 'has-error'
 							}
 							help={ dateOnSaleFromGmtValidationError as string }
+							onBlur={ validateDateOnSaleFromGmt }
 						/>
 					</div>
 
 					<div className="wp-block-column">
 						<DateTimePickerControl
+							// ref={ dateOnSaleToGmtRef }
```

Comment:
And here.

Reply:
That is because the DateTimePickerControl is not ForwardedRef yet. That is going to be covered in [#38248](https://github.com/woocommerce/woocommerce/issues/38248).